Automattic/simplenote-ios
# Note List Cell: New Metrics!

### Details:
In this PR we're several metrics affecting the Note Cell:

- The Accessory Icons's height now is dynamically adjusted to match the Title's Height. Capped between (16, 24) points
- We're now splitting Title and Body in two TextViews. This gives us a granular control over the number of lines in the Title / Body
- Several in-cell metrics have been fine tuned to match the mockups
- YES StackViews **all** over. Absolutely beautiful!!!

@aerych promise... last one regarding the Notes Cell!!!
**THANKYOU!!!**

Ref. #408

### Testing:
**Please** perform the following steps in these devices:
- [x] iPhone Xs
- [x] iPad Fullscreen (Portrait)
- [x] iPad Fullscreen (Landscape)
- [x] iPad Multitasking 1/3 (Landscape)
- [x] iPad Multitasking 1/2 (Landscape)
- [x] iPad Multitasking 2/3 (Landscape)

1. Log into Simplenote
2. Pin a note
3. Share a note
4. Verify the Note Row displays the icons on the right
5. Adjust the Font Size to the minimum, and verify the note row looks good
6. Repeat (5) but set it to the maximum
7. Please toggle Dark Mode and verify nothing breaks!

cc @SylvesterWilmott sir, mind taking a peek at the screenshots?

---

![Regular](https://user-images.githubusercontent.com/1195260/65621036-588aea00-df99-11e9-961f-1a8bf087fdbd.png)
![Min](https://user-images.githubusercontent.com/1195260/65621039-588aea00-df99-11e9-82dc-3d130c565322.png)
![Max](https://user-images.githubusercontent.com/1195260/65621041-59238080-df99-11e9-8582-e54bd49d78f5.png)
![iPad Portrait](https://user-images.githubusercontent.com/1195260/65621042-59238080-df99-11e9-8268-7df7ca4e6c62.png)
![iPad Landscape](https://user-images.githubusercontent.com/1195260/65621044-59238080-df99-11e9-9d53-51c387e4ac73.png)
![Dark](https://user-images.githubusercontent.com/1195260/65621045-59238080-df99-11e9-8c93-f74ac147f629.png)
